app/helpers/logs_helper.rb

```diff
@@ -20,6 +29,13 @@ def search_label_for_controller(controller)
     end
   end
 
+  def search_label_for_action(action_name)
+    case log_type_for_action(action_name)
```

Comment:
Could we not have case action_name directly here?

Reply:
Ah good shout